recon_service_node:
    gridm: 1.2
    holemaxsize: 50
    output_folder: $(find map_builder)/tmp_reconstructions/
    sample: 1.0
    texturesize: 320.0
    trim: 8.0
    remove_borders_iterations: 1.0
    